notebook.go

```diff
@@ -1,12 +1,39 @@
-package service
+package influxdb
 
 import (
 	"context"
+	"fmt"
 	"time"
 
 	"github.com/influxdata/influxdb/v2/kit/platform"
+	"github.com/influxdata/influxdb/v2/kit/platform/errors"
 )
 
+var (
```

Comment:
These errors were previously in a separate file, but are now included in the top-level service file similar to how other top-level services define their errors.